.github/workflows/static_code_analysis.yaml

```diff
@@ -15,24 +15,14 @@ jobs:
       - uses: pre-commit/action@v3.0.0
         env:
           SKIP: no-commit-to-branch
+      - name: ruff
+        uses: chartboost/ruff-action@v1
       - name: black
         run: |
           black --check .
-      - name: isort
-        run: |
-          isort --check --profile black .
       - name: mypy
         run: |
           mypy .
-      - name: pylint
-        run: |
-          pylint "${{ steps.common.outputs.MAIN_SRC_DIR }}"
-      - name: flake8
-        run: |
-          flake8 --config .flake8
-      - name: bandit
```

Comment:
Is ruff also implementing the checks of bandit?

Reply:
Yes, through implementing the rules of flake8-bandit.

Reply:
Nice, getting more and more convinced of ruff, hope it also works as expected